### ISSUE TITLE
smackdown vs raw 2008(BLUS30082) graphical issue

### ISSUE THREAD

**Issue by smacklover** [RPCS3/rpcs3#11637]
![glitch](https://user-images.githubusercontent.com/101231083/157635494-5e1a6b2b-bc80-47e2-8e31-35933db56553.png)

RPCS3 log is here [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/8222173/RPCS3.log.gz)

Hello there

I have a graphical problem about smackdown vs raw 2008 (a.k.a svr 2008)

I was surprised that it went better than I expected. ingame was maintained 30fps well. However, as shown in the attached screenshot, graphics are horrible and AA doesn't work 

It's frustrating because any setting doesn't work. GPU driver surely up-to-date, RPCS3 is default setting

Please help

<My Specs>

OS     Windows 10 Pro 64bit 21H2

CPU    Intel Core i7 8750H 2.20Ghz

RAM    16GB DDR4

GPU    Nvidia Geforce GTX 1060 6GB

RPCS3 Ver.    0.0.21-13343

SVR 2008 version   BLUS30082(USA) and i already tried BLES00136 

 

P.S. My sentence is not good because my English is poor :(

**Comment by digant73** [comment]
@smacklover is that still an issue?

**Comment by Satan86** [comment]
yes, the game uses some weird image reconstruction technique that rpcs3 doesn't seem to work well with, increasing resolution scale makes it look good, but native still looks like crap

**Comment by smacklover** [comment]
> [@smacklover](https://github.com/smacklover) is that still an issue?

switch anti-aliasing off can solve blurry graphic issue

**Comment by smacklover** [comment]
I haven't try that. I think rpcs3 AA is not compatible with Smackdown series

**Comment by digant73** [comment]
please provide an update on this issue
